omero/sysadmins/unix/install-web/install-wsgi.txt

```diff
@@ -0,0 +1,105 @@
+OMERO.web WSGI deployment
+=========================
+
+.. _wsgi_configuration:
+
+WSGI Configuration (Unix/Linux)
+-------------------------------
+
+If you have installed Nginx or Apache OMERO can automatically generate a
+configuration file for your web server, see :ref:`apache_wsgi_configuration`
+or :ref:`nginx_gunicorn_wsgi_configuration`.
+The location of the file will depend on your system, please refer to your
+web server's manual. See :ref:`customizing_your_omero_web_installation_unix`
+for additional customization options.
+
+.. _apache_wsgi_configuration:
+
+Apache 2.2+ configuration (mod_wsgi)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Install `mod\_wsgi <https://www.modwsgi.org/>`_.
+
+Set the following:
+
+::
+
+    $bin/omero config set omero.web.application_server "wsgi"
+
+To create a site configuration file for inclusion in the main Apache
+configuration redirect the output of the following command into a file:
+
+::
+
+    $ bin/omero web config apache-wsgi
+
+.. literalinclude:: apache-wsgi-omero.conf
+   :lines: 55-
+
+.. note::
+    To configure an HTTPS server follow
+    `the Apache documentation <http://httpd.apache.org/docs/trunk/mod/mod_ssl.html>`_.
+
+Then reload apache.
+
+
+.. _nginx_gunicorn_wsgi_configuration:
+
+Nginx Gunicorn configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set the following:
+
+::
+
+    $bin/omero config set omero.web.application_server "wsgi-tcp"
+
+
+To create a site configuration file for inclusion in a system-wide nginx
+configuration redirect the output of the following command into a file:
+
+::
+
+    $ bin/omero web config nginx-wsgi
+
+.. literalinclude:: nginx-wsgi-omero.conf
+
+.. note::
+    To configure an HTTPS server follow
+    `the nginx documentation <http://nginx.org/en/docs/http/configuring_https_servers.html>`_.
+
+.. note::
+    OMERO.web requires ``body_in_file_only`` adjusted in your default nginx 
```

Comment:
As above, I think it would be nicer to move this info into the body text and not have two noteboxes right next to each other